infra/resources/prod/main.tf

```diff
@@ -232,3 +232,17 @@ module "cosmos" {
     instance_number = "01"
   }
 }
+
+module "io_com_storage_account" {
+  source              = "../_modules/com_storage_account/"
+  resource_group_name = data.azurerm_resource_group.itn_messages.name
+  environment = {
+    prefix          = local.prefix
+    env_short       = local.env_short
+    domain          = local.domain
+    location        = local.location
+    instance_number = "01"
```

Comment:
`instance_number` is a detail that can be defined inside the module

Reply:
[3d45663](https://github.com/pagopa/io-messages/pull/107/commits/3d45663e42a77af74c6591ed4cf6f59b2e599250)